Installation/CMakeLists.txt

```diff
@@ -1178,38 +1089,36 @@ You must disable CGAL_ENABLE_CHECK_HEADERS.")
   else()
     set(has_list_of_documented_headers FALSE)
   endif()
+  list(SORT list_of_documented_headers)
+  list(REMOVE_DUPLICATES list_of_documented_headers)
+
   include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/list_of_whitelisted_headers.cmake
       OPTIONAL RESULT_VARIABLE has_list_of_whitelisted_headers)
     message("list_of_whitelisted_headers: ${list_of_whitelisted_headers}")
     ## Loop on package and headers
     set(check_pkg_target_list)
-    if(POLICY CMP0057)
-      cmake_policy(SET CMP0057 NEW)
-    endif()
+    cmake_policy(SET CMP0057 NEW)
     #get build dir for removal from deps
     get_filename_component(BUILD_DIR ${CMAKE_BINARY_DIR} NAME)
     foreach (package ${CGAL_CONFIGURED_PACKAGES_NAMES})
       if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/../${package}/include)
         set(check_pkg_headers_depends "")
-        if(has_list_of_documented_headers AND POLICY CMP0057)
+        if(has_list_of_documented_headers)
           # Then recurse2
-          file(GLOB_RECURSE ${package}_HEADERS
-            RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/../${package}/include"
-            "${CMAKE_CURRENT_SOURCE_DIR}/../${package}/include/CGAL/*.h*")
-        else() # do not recurse
-          file(GLOB ${package}_HEADERS
+          file(GLOB_RECURSE ${package}_HEADERS #don't use a hard coded list because the ones for the documented headers is enough.
             RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/../${package}/include"
             "${CMAKE_CURRENT_SOURCE_DIR}/../${package}/include/CGAL/*.h*")
+        else()
+        file(GLOB ${package}_HEADERS
+                    RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}/../${package}/include"
+                    "${CMAKE_CURRENT_SOURCE_DIR}/../${package}/include/CGAL/*.h*")
         endif()
```

Comment:
I do not remember: what is the mean of that variable `has_list_of_documented_headers`?

Reply:
it means that the awk research reported things, which should always be the case.